packages/wallets/ledger-live/rollup.config.js

```diff
@@ -0,0 +1,57 @@
+import * as process from 'node:process';
+import fs from 'node:fs';
+import ts from 'typescript';
+import { defineConfig } from 'rollup';
+import del from 'rollup-plugin-delete';
+import resolve from '@rollup/plugin-node-resolve';
+import typescript from 'rollup-plugin-typescript2';
+import { babel } from '@rollup/plugin-babel';
+import svgr from '@svgr/rollup';
+
+const extensions = ['.js', '.jsx', '.ts', '.tsx', '.svg'];
+const { dependencies = {}, peerDependencies = {} } =
+  JSON.parse(fs.readFileSync('package.json', 'utf-8'));
+const commonExternal = [
+  'react/jsx-runtime',
+  // Do not include in the bundle subpath exports like:
+  /^@reef-knot\/.*/, // e.g. @reef-knot/<package>/<exports-field-entry>
+  /^reef-knot\/.*/, // e.g. reef-knot/wallets-icons/react
+];
+const external = [
+  ...commonExternal,
+  ...Object.keys({ ...dependencies, ...peerDependencies }),
+  /node_modules/
+];
+const isDevMode = process.env.dev === 'on';
+
+export default defineConfig({
```

Comment:
Can we DRY this config rexproting(and extending if needed) from common config in the root of the project?
Similar to this https://github.com/lidofinance/lido-ethereum-sdk/blob/main/tsconfig.base.json

Reply:
This is legacy approach and we should definitely DRY it, but previously there were some issues with it, which I don't remember in details now.

Reply:
Checked, that we already have a task for this